src/__tests__/patients/care-goals/ViewCareGoal.test.tsx

```diff
@@ -0,0 +1,48 @@
+import { mount } from 'enzyme'
+import { createMemoryHistory } from 'history'
+import React from 'react'
+import { act } from 'react-dom/test-utils'
+import { Router, Route } from 'react-router-dom'
+
+import CareGoalForm from '../../../patients/care-goals/CareGoalForm'
+import ViewCareGoal from '../../../patients/care-goals/ViewCareGoal'
+import PatientRepository from '../../../shared/db/PatientRepository'
+import Patient from '../../../shared/model/Patient'
+
+describe('View Care Goal', () => {
+  const patient = {
+    id: '123',
+    givenName: 'given Name',
+    fullName: 'full Name',
+    careGoals: [{ id: '123', description: 'some description' }],
+  } as Patient
+
+  const setup = async () => {
+    jest.spyOn(PatientRepository, 'find').mockResolvedValue(patient)
+    const history = createMemoryHistory()
+    history.push(`/patients/${patient.id}/care-goals/${patient.careGoals[0].id}`)
+
+    let wrapper: any
+    await act(async () => {
+      wrapper = await mount(
+        <Router history={history}>
+          <Route path="/patients/:id/care-goals/:careGoalId">
+            <ViewCareGoal />
+          </Route>
+        </Router>,
+      )
+    })
+
+    wrapper.update()
+
+    return { wrapper }
```

Comment:
Change to `return { wrapper: wrapper as ReactWrapper }` so we can get type support

Reply:
Ok